dagger-core/src/main/java/io/odpf/dagger/core/metrics/reporters/statsd/manager/MeasurementManager.java

```diff
@@ -0,0 +1,9 @@
+package io.odpf.dagger.core.metrics.reporters.statsd.manager;
+
+import io.odpf.dagger.core.metrics.reporters.statsd.tags.StatsDTag;
+
+import java.io.Serializable;
+
+public interface MeasurementManager extends Serializable {
```

Comment:
This interface serves as an abstraction for registering and memoizing statsd tags of a component ( SplitAssigner, Parquet Reader, etc) to all measurements which get emitted from that component. Since, StatsDReporter expects tags to be sent along with each measurement call ( gauge, counter, etc), which is unnecessary when all the measurements that belong to a class have the same tags. (Not to be confused with global tags which we register during the creation call of statsdreporter itself)

This interface is not unused. It's implemented by all the 3 measurement managers: DaggerGaugeManager, DaggerHistogramManager and DaggerCounterManager.